docs/dev/table/hive/index.md

```diff
@@ -78,8 +78,8 @@ To integrate with Hive, users need the following dependencies in their project.
 
 <dependency>
   <groupId>org.apache.flink</groupId>
-  <artifactId>flink-shaded-hadoop-2-uber-2.7.5-{{site.version}}</artifactId>
-  <version>{{site.version}}</version>
+  <artifactId>flink-shaded-hadoop-2-uber</artifactId>
+  <version>2.7.5-7.0</version>
```

Comment:
Can we auto-generate the shaded version? It's hard to trace the shaded dependency version once the shaded version is bumped.

Reply:
Thank you, but I don't quite understand what you mean. I want to add it to the "docs/_config.yml" file and use the version number as a variable.

Reply:
Whoever bumps the flink-shaded version also has to ensure that the documentation is up-to-date.
While it _is_ possibly to extract this property from maven, it doesn't seem worth the trouble. (You'd have to setup something similar to the REST API docs; a generator populates some field in _config.yml, and a test verifies that the value is identical to the current property)

Reply:
In any case, such a setup would be out-of-scope of this PR.

@yangjf2019 Having a variable in docs/_config.yml for the flink-shaded version sounds like a good idea.

Reply:
I think @zentol proposed a good idea. We can create a followup JIRA issue to track this effort. 
And having a flink-shaded version in `docs/_config.yml` as a temporary solution in this PR.

Reply:
Hi, @wuchong @zentol , Thank you very much ! Someone has committed for this [PR](https://github.com/apache/flink/commit/614e9b73345c3ec09d55f75426968b7329c7953a),Maybe i will closed .

Reply:
However, it doesn't fix the shaded version. We still need a pull request to fix it.

cc @bowenli86

Reply:
Hi, @wuchong @zentol ,I compiled the latest code, however we do not find the maven dependency as follows. 
`<dependency>
  <groupId>org.apache.flink</groupId>
  <artifactId>flink-shaded-hadoop-2-uber</artifactId>
  <version>2.6.5-1.10-SNAPSHOT</version>
  <scope>provided</scope>
</dependency>
`
`<dependency>
  <groupId>org.apache.flink</groupId>
  <artifactId>flink-hadoop-compatibility_1.10-SNAPSHOT</artifactId>
  <version>1.10-SNAPSHOT</version>
  <scope>provided</scope>
</dependency>`
so I think this [PR](https://github.com/apache/flink/commit/614e9b73345c3ec09d55f75426968b7329c7953a) is problematic. Many users will ask why this maven dependency is not found.And I saw that [‘travis-ci’](https://travis-ci.org/apache/flink/builds/578111960?utm_source=github_status&utm_medium=notification) has failed to build. Why can it be merged into master?
I have submitted a new [Pull Request](https://github.com/apache/flink/pull/9591), please take a look, thank you!